relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -719,16 +722,21 @@ mod tests {
                     }
                 }
 
-                // Normalize first, to make sure that all things are correct as in the real pipeline:
-                let res = normalize_event(
+                // Validate and normalize first, to make sure that all things
+                // are correct as in the real pipeline:
+                let res = validate_transaction(&mut event, &TransactionValidationConfig::default());
+                assert!(res.is_ok());
+                let res = validate_event_timestamps(&mut event, &EventValidationConfig::default());
+                assert!(res.is_ok());
```

Comment:
```suggestion
                // Validate and normalize first, to make sure that all things
                // are correct as in the real pipeline:
                let res = validate_transaction(&mut event, &TransactionValidationConfig::default());
                assert!(res.is_ok());
                let res = validate_event_timestamps(&mut event, &EventValidationConfig::default());
                assert!(res.is_ok());
```

This should not be necessary for the tests that use this macro. Ideally we would even call only `extract_tags` here.

Reply:
Removed validation calls, but not replacing with `extract_tags` as I don't want to overload this PR.